lib/ansible/plugins/action/__init__.py

```diff
@@ -673,6 +673,12 @@ def _fixup_perms2(self, remote_paths, remote_user=None, execute=True):
 
         if res['rc'] == 0:
             return remote_paths
+        if res['rc'] == 127:
```

Comment:
most of these the user cannot do much about, this would be better as a debug:
```
self._display.debug(f'Failed to set facl {setfacl_mode}, got:{res!r}')
```